database/migrations/000005_create_transactions_table.up.sql

```diff
@@ -1,20 +1,21 @@
-CREATE TABLE `transactions`
+CREATE TABLE transactions
 (
-    `id`                 BIGINT UNSIGNED NOT NULL AUTO_INCREMENT,
-    `accepting_block_id` BIGINT UNSIGNED NULL,
-    `transaction_hash`   CHAR(64)        NOT NULL,
-    `transaction_id`     CHAR(64)        NOT NULL,
-    `lock_time`          BIGINT UNSIGNED NOT NULL,
-    `subnetwork_id`      BIGINT UNSIGNED NOT NULL,
-    `gas`                BIGINT UNSIGNED NOT NULL,
-    `payload_hash`       CHAR(64)        NOT NULL,
-    `payload`            BLOB            NOT NULL,
-    `mass`               BIGINT          NOT NULL,
-    `version`            INT             NOT NULL,
-    PRIMARY KEY (`id`),
-    UNIQUE INDEX `idx_transactions_transaction_hash` (`transaction_hash`),
-    INDEX `idx_transactions_transaction_id` (`transaction_id`),
-    CONSTRAINT `fk_transactions_accepting_block_id`
-        FOREIGN KEY (`accepting_block_id`)
-            REFERENCES `blocks` (`id`)
+    id                 BIGSERIAL,
+    accepting_block_id BIGINT NULL,
+    transaction_hash   CHAR(64) NOT NULL,
+    transaction_id     CHAR(64) NOT NULL,
+    lock_time          BYTEA NOT NULL,
+    subnetwork_id      BIGINT CHECK (subnetwork_id >= 0) NOT NULL,
```

Comment:
I think we are missing foreign key for subnetwork_id.
And the moment you have the foreign key, it seems like the check constraint becomes redundant.

Reply:
Fixed, thanks